src/modules/Kdump.rb

```diff
@@ -1145,7 +1145,7 @@ def crash_kernel_values
       # modification.
       # The old value (ensuring the Array format) will be returned.
       if @crashkernel_list_ranges
-        return Array(@crashkernel_param_values.dup)
+        return Array(@crashkernel_param_values.to_s.dup)
```

Comment:
Sorry, I wrongly assumed `@crashkernel_param_values` was string or symbol, but it can also be Array